.github/build.yaml.gomplate

```diff
@@ -249,11 +248,11 @@
       - name: Build ISO from local build 🔧
         if: github.event_name != 'schedule' && !startsWith(github.ref, 'refs/tags/')
         run: |
-          sudo -E MAKEISO_ARGS="--name cOS-Seed-{{{ $flavor }}}-${{ env.COS_VERSION }}-{{{ $config.arch }}}" make local-iso
+          sudo -E MAKEISO_ARGS="--name cOS-Seed-{{{ $flavor }}}-${{ env.COS_VERSION }}-{{{ $config.arch }}}{{{- if eq $config.arch "arm64"}}} --repo ${{ env.FINAL_REPO }}{{{end}}}" make local-iso
       - name: Build ISO from remote repositories 🔧
         if: github.event_name == 'schedule' || startsWith(github.ref, 'refs/tags/')
         run: |
-          sudo -E MAKEISO_ARGS="--name cOS-Seed-{{{ $flavor }}}-${{ env.COS_VERSION }}-{{{ $config.arch }}}" make iso
+          sudo -E MAKEISO_ARGS="--name cOS-Seed-{{{ $flavor }}}-${{ env.COS_VERSION }}-{{{ $config.arch }}}{{{- if eq $config.arch "arm64"}}} --repo ${{ env.FINAL_REPO }}{{{end}}}" make iso
```

Comment:
if we add back the final repo here aren't we back at https://github.com/rancher-sandbox/cOS-toolkit/pull/1289 ?